packages/epo-widget-lib/src/charts/Tooltip/index.tsx

```diff
@@ -94,9 +95,17 @@ const Tooltip: FunctionComponent<PropsWithChildren<TooltipProps>> = ({
       {typeof x === "number" && typeof y === "number" && (
         <Styled.Tooltip
           className={className}
-          style={{
-            transform: `translate(calc(${xOffset} + ${x}px), calc(${yOffset} + ${y}px))`,
-          }}
+          style={
+            isSafari()
```

Comment:
`transform` doesn't work cleanly in Safari, instead do a combo of left/right and a translate to try and clean up the final positioning. The result is slightly less nice than Chrome and Firefox but very much functional